eng/pipelines/templates/stages/archetype-spec-gen-sdk.yml

```diff
@@ -200,6 +200,17 @@ extends:
                 $optional_params
             displayName: 'Generate SDK'
 
+          - task: Powershell@2
+            inputs:
+              filePath: $(SdkRepoDirectory)/eng/common/scripts/Save-Package-Properties.ps1
+              arguments: >
+                -ServiceDirectory $(GeneratedSDK.ServiceName)
```

Comment:
Where is the ServiceName set? Also do we want to scope to an entire service directory or should we scope to individual packages?

Reply:
It's set in the `spec-gen-sdk` tool. We have a [pending work item](https://github.com/Azure/azure-sdk-tools/issues/10127) to move the settings to the runner. Supporting multiple packages in API view will be a separate task.

Reply:
My question is more around if we are only changing one package in a service directory will this cause us to create an APIView for all the packages in that directory?

Reply:
@weshaggard  We are even planning to support multiple services, so scope is going to be even larger.

Reply:
I just want to make sure we are only generating apiview for the sdks that change.